source/ShipEvent.h

```diff
@@ -43,6 +43,7 @@ class ShipEvent {
 		// of that ship's government; this will result in temporary animosities
 		// between the two governments. If a ship is "forbearing," it can only
 		// be "provoked" if its shields are below 90%.
+		// Some governments are provoked by starting a scan.
```

Comment:
This new government attribute makes it so that a ship becomes hostile if a scan has started, but how about making it so that a government that has a negative reputation hit for completing a scan on their ships also get provoked?

If you look at the other ship events, we have assist only being able to be done on friendly ships, disable, board, capture, and destroy only being able to be done after a ship has been provoked, and atrocity caused instant negative rep and therefore hostility. But with the new scan penalties added, should a government have a negative penalty for completing a scan, they don't actually get provoked by being scanned, so how is the player to know that they don't want to get scanned?

Reply:
Isn't that already a problem? Like if a government doesn't like being assisted, you get a negative rep hit without provoking the government.

Reply:
Technically yes, but I'm thinking more about behavior under normal circumstances. (I wouldn't consider a faction that doesn't like to be assisted to be normal circumstances.) So what we could do instead to cover all our bases is have any negative rep hit also count as provocation, although at that point we should save that for another PR since it doesn't have to deal with scanning anymore.